acvm/src/lib.rs

```diff
@@ -233,21 +235,25 @@ pub fn hash_constraint_system(cs: &Circuit) -> [u8; 32] {
 // This is set to match the previous functionality that we had
 // Where we could deduce what black box functions were supported
 // by knowing the np complete language
-pub fn default_is_black_box_supported(
+pub fn default_is_opcode_supported(
     language: Language,
-) -> compiler::transformers::IsBlackBoxSupported {
+) -> compiler::transformers::IsOpcodeSupported {
     // R1CS does not support any of the black box functions by default.
     // The compiler will replace those that it can -- ie range, xor, and
-    fn r1cs_is_supported(_opcode: &BlackBoxFunc) -> bool {
+    fn r1cs_is_supported(_opcode: &Opcode) -> bool {
         false
     }
 
     // PLONK supports most of the black box functions by default
```

Comment:
same here re black box function being changed to opcode

Reply:
Done